alibi/explainers/anchor_base.py

```diff
@@ -24,7 +25,7 @@ def __init__(self, samplers: List[Callable], **kwargs) -> None:
         """
 
         self.sample_fcn = samplers[0]
-        self.samplers = None  # type: List[Callable]
+        self.samplers = None  # type: Optional[List[Callable]]
```

Comment:
This is an unfortuante exception in the tirade of the original comment because the class `AnchorBaseBeam` is inherited by `DistributedAnchorBaseBeam` which uses a `self.samplers` attribute even though it is never set for this base class. Sorting this out would require more refactoring than necessary for this PR.